src/docs/glossary.md

```diff
@@ -4,68 +4,94 @@ eleventyNavigation:
   key: Glossary
 ---
 
-# Glossary and Buzzwords
+# Glossary
+
+This page provides two lists of terms — one for [Eleventy-specific terminology](#eleventy-specific-terminology) that may be useful for when building out a project using Eleventy, and one for [industry jargon](#industry-terms-and-buzzwords) that may be useful for understanding context.
+
+_Work in progress: {% include "edit-on-github.njk" %}_
+
+## Eleventy-Specific Terminology
+
+### Template
+
+A content file written in a [format such as Markdown, HTML, Liquid, Nunjucks, and more](/docs/languages/), which Eleventy transforms into a page (or pages) in the built site. Templates can access [data](#data) exposed through the [data cascade](#data-cascade) with templating syntax.
+
+### Layout
+
+A template which wraps around another template, typically to provide the scaffolding markup for content to sit in.
+
+[Read more about using layouts.](/docs/layouts/)
+
+### Data
+
+Variables that can be used inside [templates](#template) and [layouts](#layout) using templating syntax. The data for a given template is aggregated through a process called the [data cascade](#data-cascade).
+
+### Data Cascade
+
+Eleventy's order of operations for evaluating all [data](#data) for any given [template](#template), and for resolving conflicts that arise. The data cascade follows the principle of colocation, so data defined broadly to apply to many templates will be overruled by data that targets the given template more specifically.
+
+[Read more about the data cascade.](/docs/data-cascade/)
+
```

Comment:
Possible other terms: pagination, collection, plugin, serverless, filter, shortcode

Reply:
Hey, @5t3ph! I've added definitions for the other terms you've listed. Would you be willing to take a look and let me know what you think?